CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+
+First off, thanks for taking the time!
+If you are interested in contributing to Opta, we'd love to hear from you! Drop us a line in our [Slack](https://slack.opta.dev/).
+
+## Orientation
+
+Here's a list of repositories that contain Opta-related code:
+
+- [opta](https://github.com/run-x/opta)
+  is the main repository containing the `Opta` core code and modules.
+- [opta-docs](https://github.com/run-x/opta-docs) contains all the documentation and the Opta installation script.
+
+## Types of Contributions
```

Comment:
In the future, we could have a section for custom module here?

Reply:
💯 %